pkgs/applications/misc/st/default.nix

```diff
@@ -17,11 +17,12 @@ in stdenv.mkDerivation rec {
   configFile = optionalString (conf!=null) (writeText "config.def.h" conf);
   preBuild = optionalString (conf!=null) "cp ${configFile} config.def.h";
 
-  nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ libX11 ncurses libXext libXft fontconfig ] ++ extraLibs;
+  nativeBuildInputs = [ pkgconfig makeWrapper ];
+  buildInputs = [ libX11 ncurses libXext libXft fontconfig dmenu ] ++ extraLibs;
```

Comment:
Arf, sorry, one last thing... I think you don't need to add it in `buildInputs` since you add it in the `PATH`.

Reply:
My fault, should be fixed now.